lesson_09/homeworks.py

```diff
@@ -0,0 +1,69 @@
+
+""" Задача - надрукувати табличку множення на задане число, але
+лише до максимального значення для добутку - 25.
+Код майже готовий, треба знайти помилки та випраавити\доповнити.
+"""
+
+def multiplication_table(number=None):
+    if number is None:
+        return ("The input must be a positive integer.")
+    if not isinstance(number, int):
+        return ("The input must be a positive integer.")
+    if number < 1:
+        return ("The input must be a positive integer.")
```

Comment:
```suggestion
        return ("The input must be a positive integer.")
    if not isinstance(number, int):
        return ("The input must be a positive integer.")
    if number < 1:
        return ("The input must be a positive integer.")
```
Це все - рейз помилок

Reply:
```suggestion
    if number is None or not isinstance(number, int):
        raise TypeError("The input must be a positive integer.")
    if number < 1:
        raise ValueError("The input must be a positive integer.»)
```

Reply:
нижче новий комент, бо заміняю більше строк ніж ви запропонували.